etc/meteo-qt.profile

```diff
@@ -0,0 +1,43 @@
+# Firejail profile for meteo-qt
```

Comment:
Add a description:
`# Description: System tray application for weather status information`